plugins/bitwarden/plugin.spec.yaml

```diff
@@ -94,6 +122,7 @@ types:
         - 1-Admin
         - 2-User
         - 3-Manager
+        - 4-Custom
```

Comment:
If this is an output type, should we forgo enums entirely? Thinking we usually do to avoid compatibility issues if anything in the service changes.

Reply:
Yeah I wasn't sure why if we have specified an output type of string if it needed these enum values? I probably should remove this then?

Reply:
Yeah I think unless it's used in an input which i can't see. We could maybe add the values into the description if we want to give users an idea of what the output would be.

Reply:
I did decide to keep these enums on the plugin spec but I've removed the number prefix. We have an input enum elsewhere and though it made sense to keep here?